### ISSUE TITLE
investigate failure of a11y snapshot action

### ISSUE THREAD

**Issue by gbinal** [GSA/site-scanning#1737]
https://github.com/GSA/site-scanning-engine/actions/workflows/create-a11y-snapshot.yml

**Comment by gbinal** [comment]
FYI, in case it's related - https://github.com/GSA/site-scanning/issues/1758

**Comment by gbinal** [comment]
This may be fixed, esp. now that it's on a new schedule. Let's monitor it for a few days then we can close.

**Comment by gbinal** [comment]
again, great work. done.
